pkgs/by-name/op/openfortivpn-webview-qt/package.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchFromGitHub, qt6, }:
+stdenv.mkDerivation rec {
+  pname = "openfortivpn-webview-qt";
+  version = "v1.1.2-electron";
+
+  src = fetchFromGitHub
+    {
+      owner = "gm-vm";
+      repo = "openfortivpn-webview";
+      rev = version;
+      hash = "sha256-BNotbb2pL7McBm0SQwcgEvjgS2GId4HVaxWUz/ODs6w=";
+    };
+  sourceRoot = "source/openfortivpn-webview-qt";
+
+
+  nativeBuildInputs = [ qt6.wrapQtAppsHook qt6.qtwebengine qt6.qmake ];
+  buildPhase = ''
+    qmake .
+    make
+  '';
+  installPhase = ''
+    mkdir -p $out/bin
+    cp openfortivpn-webview $out/bin/
+  '';
+  meta = with lib; {
+    description = "Application to perform the SAML single sing-on and easily retrieve the SVPNCOOKIE needed by openfortivpn.";
+    homepage = "https://github.com/gm-vm/openfortivpn-webview/tree/main";
+    license = licenses.mit;
+    maintainers = [ "jonboh" ];
```

Comment:
please add yourself to the main maintainers list in a separate first commit, then use `lib.maintainers.jonboh` to refer to your maintainers entry. CONTRIBUTING.md has more info on this.

Reply:
sorry, missed that part and the commit convention on the CONTRIBUITING.md, btw thanks for the review! :)